client/src/app/site/motions/modules/motion-poll/motion-poll-detail/motion-poll-detail.component.ts

```diff
@@ -103,6 +158,34 @@ export class MotionPollDetailComponent extends BaseViewComponent implements OnIn
         }
     }
 
+    public getResult(): { name: string; value: number }[] {
+        const model = this.poll.getModel();
+
+        return [
+            ...Object.entries(model.options[0])
```

Comment:
Does this work for `YN` pollmethod? It seems, that the `A` value is also copied. I would suggest to be more explicit here and just give `Y`, `N` and conditionally `A` in the array.